templates/uefi/generic-qemu-uefi-boot-template.jinja2

```diff
@@ -0,0 +1,62 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% set ctx_arch = arch %}
+{% if arch == "arm" %}
+{% set ctx_arch = 'arm64' %}
+{% set ctx_cpu = 'cortex-a15' %}
+{% set console_dev = 'ttyAMA0' %}
+{% endif %}
+{% if arch == "arm64" %}
+{% set ctx_arch = 'arm64' %}
+{% set console_dev = 'ttyAMA0' %}
+{% endif %}
+{% if arch == 'x86_64' %}
+{% set ctx_arch = 'amd64' %}
+{% set console_dev = 'ttyS0' %}
+{% endif %}
+{% if arch == 'i386' %}
+{% set ctx_arch = 'i386' %}
+{% set ctx_cpu = 'host' %}
+{% set console_dev = 'ttyS0' %}
+{% endif %}
+{% block metadata %}
+{{ super() }}
+{% endblock %}
+{% block main %}
+{{ super() }}
+{% endblock %}
+{% block actions %}
+context:
```

Comment:
This will conflict with the other changes to make the context more flexible, see #108.  Some rebase will be needed, either this PR or the other one depending on which one gets merged first.  It's OK to leave it as-is for now to test it on staging.

Reply:
@roxell Please rebase this PR now that #108 has been merged.